### DISCUSSION TITLE
Possible to call an engine from SurfingKeys ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [hoothin/SearchJumper#183] [Q&A]
With SurfingKeys, arbitrary JS code can be run (maybe with limitations, I don’t know yet). This gave me the idea to maybe use it call some engine from SearchJumper ? Would it be possible ?

**Comment by hoothin** [comment]
Yes, there are two ways.
1.  Try to run JS code `document.dispatchEvent(new CustomEvent("searchJumper", {detail: {action: "search", name: "Your Engine Name"}}));`
2. Navigate to `https://search.hoothin.com/all?kw=Your Keywords&engine=Your Engine Name`
